main.go

```diff
@@ -128,6 +128,11 @@ moreLoop:
 			"repocursor":  cursor,
 			"querystring": githubv4.String(queryString),
 		}
+
+		if page > 1 {
+			time.Sleep(1 * time.Second)
+		}
```

Comment:
can you summarize the reason behind this sleep in a comment above this bit of code?

the commit message is great and should stay that way, but people reading the code won't be looking at the git log, so an FYI here is perhaps more important.